src/main/scala/com/databricks/labs/overwatch/ApiCall.scala

```diff
@@ -132,12 +138,7 @@ class ApiCall(env: ApiEnv) extends SparkSessionWrapper {
     logger.log(Level.INFO, s"Loading $req -> query: $getQueryString")
     try {
       val result = Http(req + getQueryString)
-        .headers(Map[String, String](
-          "Content-Type" -> "application/json",
-          "Charset"-> "UTF-8",
-          "User-Agent" -> s"databricks-labs-overwatch/${env.packageVersion}",
-          "Authorization" -> s"Bearer ${env.rawToken}"
-        ))
+        .copy(headers = httpHeaders)
```

Comment:
why not construct `HttpRequest` directly instead of creating it indirectly & then copying? https://github.com/scalaj/scalaj-http/blob/master/src/main/scala/scalaj/http/Http.scala#L791

Reply:
It's a bug in scalaJ. I tested several different options and this was the one that worked. See the bug below.
https://github.com/scalaj/scalaj-http/issues/113